engine/dlib/src/dmsdk/dlib/profile.h

```diff
@@ -110,7 +108,7 @@
  *
  * ```cpp
  * DM_PROPERTY_EXTERN(rmtp_GameObject);
- * DM_PROPERTY_U32(rmtp_ComponentsAnim, 0, FrameReset, "#", &rmtp_GameObject);
+ * DM_PROPERTY_U32(rmtp_ComponentsAnim, 0, PPF_FRAME_RESET, "#", &rmtp_GameObject);
```

Comment:
What's the F in PPF? Frame?

Reply:
PROFILE_PROPERTY_FLAG_

I don't like to shorten enums (because they become cryptic).
I guess I should just write it out....